### ISSUE TITLE
Grammar is wrong for make

### ISSUE THREAD

**Issue by z5267282** [z5267282/mve#114]
If you run

```sh
python3 make.py
```

You should have to enter all 3 paths and so the prompt should be

```
enter ... for these paths
```

rather than

```
... for this path
```

this is because of how the counting works (I think none should be 1) in `video_paths.py`.

```py
    @staticmethod
    def count_none(args: list[None | str]) -> int:
        return sum(1 if arg is None else 0 for arg in args)
```

**Comment by z5267282** [comment]
This was due to the walrus operator.
If you do something like

```py
        if nones := VideoPaths.count_none(args) > 0:
```
 
 the argument becomes a `bool` not the original return type of the function (here, meant to be `int`).

**Comment by z5267282** [comment]
Fixed in 1483218
